tests/test_cms_menus.py

```diff
@@ -726,6 +839,37 @@ def test_menu_with_softroot_rendering_nested_softroot_child(self):
 
         self.assertTreeQuality(soft_root, mock_tree, 'title', 'level')
 
+    def test_draft_menu_item_atributechanges_on_published_page_menu(self):
+        """
+        Draft menu item attribute changes render changes in draft menucontent version only.
```

Comment:
duplicate "changes" in the comment

```suggestion
        A published page only shows published menu when a draft version exists with changes to the attributes.
```

Reply:
Updated